test/integration/image-component/default/pages/missing-src.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Image from 'next/image'
+
+const Page = () => {
+  return (
+    <div>
+      <p>Hello World</p>
+      <Image id="unsized-image" width={200}></Image>
```

Comment:
This image is not using the unsized property but the id makes it look like it is